src/main/java/com/scg/stop/domain/gallery/domain/Gallery.java

```diff
@@ -29,28 +28,40 @@ public class Gallery extends BaseTimeEntity {
     @Column(nullable = false)
     private String title;
 
-    @Column(nullable = false, columnDefinition = "TEXT")
-    private String content;
-
     @Column(nullable = false)
     private Integer year;
 
     @Column(nullable = false)
     private Integer month;
 
+    @Column(nullable = false)
+    private Integer hitCount = 0;
+
     @OneToMany(fetch = LAZY, mappedBy = "gallery")
     private List<File> files = new ArrayList<>();
```

Comment:
현재 `deleteGallery` API에서 외래키 제약 조건 때문에 파일과 연관관계를 가지는 갤러리가 삭제되지 않는 것 같습니다. 아래와 같이 수정해서 cascade 삭제가 가능하도록 변경해야할 것 같습니다.
```suggestion
    @OneToMany(fetch = LAZY, mappedBy = "gallery", cascade = CascadeType.REMOVE)
    private List<File> files = new ArrayList<>();
```

Reply:
감사합니다!!
[ef66c6b](https://github.com/SystemConsultantGroup/s-top-backend/pull/33/commits/ef66c6b5efd017299bec29536202e8c5887d154b)